### PR TITLE
Migrate in exec to v0.14

### PR DESCRIPTION
**Pull request by cosmo0920**

### CODE REVIEW

File: lib/fluent/plugin/in_exec.rb [hunk 1]
```diff
@@ -65,15 +67,15 @@ def configure(conf)
...
+        raise Fleunt::ConfigError, "'tag' or 'tag_key' option is required on exec input"
       end
 
       if @time_key
         if @time_format
           f = @time_format
           @time_parse_proc =
             begin
-              strptime = Strptime.new(f)
+              strptime = ::Strptime.new(f)
```
Comment (okkez) [inline lib/fluent/plugin/in_exec.rb]: Do we need full qualified class name?
I think this is not needed here.

File: lib/fluent/plugin/in_exec.rb [hunk 2]
```diff
@@ -106,37 +108,15 @@ def start
...
-
-        begin
-          Process.kill(:KILL, @pid)
-        rescue #Errno::ECHILD, Errno::ESRCH, Errno::EPERM
-        end
-        @thread.join
-      end
-
       super
     end
```
Comment (okkez) [inline lib/fluent/plugin/in_exec.rb]: We can remove `shutdown` definition.

File: lib/fluent/plugin/in_exec.rb (hunk 1 above)
Comment (cosmo0920) [inline lib/fluent/plugin/in_exec.rb]: Yes, we can remove this full qualifier.

**Comment by okkez** [comment]
In `setup`, you can use `event_time("2011-01-02 13:14:15")` instead of `Fluent::EventTime.parse("2011-01-02 13:14:15")`.

**Comment by tagomoris** [comment]
`child_process` plugin helper should this plugin code shorter.

**Comment by tagomoris** [comment]
LGTM. Thank you for great change!

**Pull request merged**